### DISCUSSION TITLE
I can't register github global campus

### DISCUSSION THREAD

**Discussion by ghost** [community/community#133812] [GitHub Education]
### Hi everyone,

It should be at this page 
<img width="595" alt="2" src="https://github.com/user-attachments/assets/ecbfd7b4-0b52-4a67-95e2-16a977777eee">

but actually is asking me to pay

**Comment by SohanKafle** [comment]
Hi [MuskZhouMQ](https://github.com/MuskZhouMQ),

It looks like you're being asked to pay instead of registering for GitHub Global Campus. Here are some quick steps to troubleshoot:

**1. Ensure you're on the correct GitHub Education page: https://education.github.com/
2. Verify you've selected the correct academic status (Student or Teacher).
3. Confirm your GitHub account email is verified.
4. Clear your browser's cache and cookies or try using an incognito window.
5. Use a different browser if the issue persists.
6. Contact [GitHub Support](https://support.github.com/) with details and screenshots if the problem continues.**

If you need further help, let me know!
